app/packs/src/components/sampleTaskInbox/SampleTaskNavigationElement.js

```diff
@@ -3,16 +3,16 @@ import { observer } from 'mobx-react';
 import { Button, Badge } from 'react-bootstrap';
 import { StoreContext } from 'src/stores/mobx/RootStore';
 
-const SampleTaskNavigationElement = ({}) => {
+function SampleTaskNavigationElement() {
```

Comment:
Why have you changed this?
`const SampleTaskNavigationElement = ({}) => ` is correct

Reply:
There were linting errors. I'll undo it